lib/osf-components/addon/components/move-file-modal/component.ts

```diff
@@ -210,7 +210,13 @@ export default class MoveFileModalComponent extends Component<MoveFileModalArgs>
     @waitFor
     async moveFile(file: File, destinationNode: NodeModel, path: string, provider: string,
         options?: { conflict: string }) {
-        return await taskFor(file.move).perform(destinationNode, path, provider, options);
+        const { requestData, status } = await taskFor(file.move).perform(destinationNode, path, provider, options);
+        let returnValue = requestData;
```

Comment:
So is there ever a chance that you'll get a return value that's a string but not an empty string? If so, this isn't going to do what you want it to.